grade-scripts/dump-requests.js

```diff
@@ -193,8 +205,11 @@ const run = async () => {
             data = { url: `http://${siteToCheck}`, failed: true }
         } else {
             console.log(chalk.green(`got ${data.requests.length} requests for ${siteToCheck}`))
+            data.hasIFrameRequests = data.requests.some((requestData) => !!requestData[2])
```

Comment:
The site JSON data will now contain frame data if a request was loaded from an iframe on the page. So something like:

```json
{
    "url": "http://something.com",
    "requests": [
        ["http://something.com/image.png", "image", null],
        ["http://tracker1.com/tracker_in_ifame.js", "script", "http://tracker.com/iframeurl"]
    ],
    "hasIFrameRequests": true
}
```

Test this against yahoo.com to see it in action.